ArduCopter/Copter.cpp

```diff
@@ -566,7 +566,9 @@ void Copter::loop_rate_logging()
 // should be run at 10hz
 void Copter::ten_hz_logging_loop()
 {
-    // log attitude data if we're not already logging at the higher rate
+    // always write AHRS attitude at 10Hz
+    ahrs.Write_Attitude(attitude_control->get_att_target_euler_rad() * RAD_TO_DEG);
```

Comment:
I think I would vote to still log `ATT` at 25hz if medium logging is selected, but it make sense to drop it from 400Hz logging.

Reply:
My understanding from the dev call was to log at 10Hz only. I'm not going to open that can of worms again.

Reply:
Yes, the consensus was that ATT should be logged at a fixed rate of 10 Hz.

The idea is to move away from the ATT message. Why do you want to log ATT faster?

Reply:
The current behaviour is to log ATT at 10 Hz if MASK_LOG_ATTITUDE_MED is on and not logging it at all if MASK_LOG_ATTITUDE_MED is off.

It looks like MASK_LOG_ATTITUDE_FAST is being used to log EKF_POS at 10 Hz.

These are a bit of a mess.

Reply:
Note that ATT is not currently logged at 25Hz. Other things are but not ATT, at least not in copter.